### ISSUE TITLE
Completions flag does not work in native-image

### ISSUE THREAD

**Issue by olafurpg** [coursier/coursier#1495]
Using the `cs` binary from https://github.com/coursier/coursier/releases/tag/v2.0.0-RC5-6
```sh
$ cs --completions zsh
Exception in thread "main" java.lang.NullPointerException
	at coursier.cli.Coursier$.zshCompletions(Coursier.scala:34)
```

**Comment by alexarchambault** [comment]
That must be because the resource [is missing from the native image](https://github.com/coursier/coursier/blob/fa4c5084ee33ba2b2896f0f52d7cf31c17dfb5d0/scripts/generate-graalvm-launcher.sh#L50).

I'm in the midst of revamping the native image generation, I'll fix that along the way.

**Comment by olafurpg** [comment]
Makes sense. The native image is otherwise working really well. Been using it for several months now
